### DISCUSSION TITLE
posts/gtkd

### DISCUSSION THREAD

**Discussion by giscus** [whexy/whexy-blog-comments#22] [General]
# posts/gtkd

苹果在 WWDC 上公布了 Game Porting Toolkit，可以将 DirectX API 转译到自家的 Metal API。通过使用套件中的模拟器，在 macOS 上运行 Windows 游戏成了现实。

https://www.whexy.com/posts/gtkd

**Comment by whexy** [comment]
游戏社区对 macOS 平台的转变感到非常兴奋！爱好者们将验证游戏的可玩性并提供错误支持。
如果在安装GPTK过程中遇到问题，可以参考：https://www.applegamingwiki.com/wiki/Game_Porting_Toolkit
